frontend/public/components/catalog/_catalog.scss

```diff
@@ -196,7 +196,7 @@ $catalog-tile-width: $co-m-catalog-tile-width;
     top: 4.75rem; // --pf-c-page__header--MinHeight
 
     .catalog-item-header-pf {
-      padding: var(--pf-global--spacer--lg) var(--pf-global--spacer--lg) var(--pf-global--spacer--md) var(--pf-global--spacer--lg);
+      padding: var(--pf-global--spacer--lg) var(--pf-global--spacer--2xl) var(--pf-global--spacer--md) var(--pf-global--spacer--lg);
```

Comment:
Not sure it's worthy of a change, but just an FYI... this padding *just* makes it on the inside of the `x` ... with the right word length / screen size... we can get quite close:

![image](https://user-images.githubusercontent.com/8126518/78059492-ee903a80-7357-11ea-80c8-eb2ffbd92bda.png)

If we added more padding it'll just take away more from the header space. Feels a little unbalanced imo.

Reply:
In `_catalog.scss` can we not simply remove this:
```
    .pf-c-title {
      margin-right: 0 !important;
    }
```

Reply:
Looks like we can remove that. I figured it was there for another version of the side-panel that used that space for a button but I cannot find the reason why that was added (as part of the move to PF4 modals).